### PR TITLE
[BIG PR] Orders

### PR DESCRIPTION
**Pull request by AELSchauer**
Hey guys, 

This creates all the foundational stuff for Orders -- migrations, tables, and models. I also added updated the seed file so it would create users and orders.

I think Amara said she created the OrderController, but the create method is more complex so I commented it out. I'll put her comments below as well as mine on that particular feature.

-Ashley

### CODE REVIEW

File: app/controllers/orders_controller.rb
Comment (AELSchauer) [inline app/controllers/orders_controller.rb:16]: Here are Amara's comments: the four lines below probably would be best delegated to a GameCreator PORO

Comment (AELSchauer) [inline app/controllers/orders_controller.rb]: Because there is now a many-to-many relationship between the donations and the orders, this create method needs some work.

Also, why was this controller created before we had the Orders table created?

**Comment by AELSchauer** [comment]
I am going to create a hound-cleanup branch to fix the violations in this code.

**Comment by AELSchauer** [comment]
To whomever approves this request... After approving, please reset the database on master and create a new commit right away in order to fix any weird migration issues.

rake db:drop:all db:create db:migrate db:seed

File: app/controllers/orders_controller.rb
Comment (TheKaterTot) [reply app/controllers/orders_controller.rb:16]: I think the GameCreator class was just an example from the tutorial, right? Otherwise, I have no idea what a GameCreator is.

File: app/models/donation.rb
Comment (TheKaterTot) [inline app/models/donation.rb:8]: Good refactor!

File: db/migrate/20170223233736_create_order_donations.rb
Comment (TheKaterTot) [inline db/migrate/20170223233736_create_order_donations.rb:6]: Is this the quantity of orders?

**Review by TheKaterTot — APPROVED** [review]

**Comment by TheKaterTot** [comment]
I tested the migrations before I merged and everything is good. I didn't even need to drop the table for it to work, which is ideal since we are aiming for production level code.

**Pull request merged**
